### DISCUSSION TITLE
Namespace

### DISCUSSION THREAD

**Discussion by Notadumdum** [miguelgrinberg/python-socketio#1078] [Q&A] [answered]
Good day... is the 'socket.io' present in this url
```wss://ws2.basqu.com/socket.io/?EIO=3&transport=websocket``` a namespace... and if I want to connect to the server should it be among like this ```https://ws2.basqu.com/socket.io/``` or like this ```https://ws2.basqu.com``` without it... which of them is the correct way...

**Comment by miguelgrinberg** [comment, marked as answer]
Normally you connect to the domain. The documentation shows examples of this, btw: https://python-socketio.readthedocs.io/en/latest/client.html#connecting-to-a-server.

**Reply by Notadumdum** [reply]
So is the/socket.io/ a namespace

**Reply by Notadumdum** [reply]
And also please could you be precise with your answer.. like could you make it relate to exactly what I asked and gave... thank you very much @miguelgrinberg

**Reply by miguelgrinberg** [reply]
No. The `/socket.io/` path is an endpoint that is internally used by Socket.IO. You as an application or server developer never need to use that.

May I suggest that you read the documentation? Both for this Python package and for the JavaScript version as well. They answer these and other basic questions about Socket.IO.
